apps/explorer/src/components/OwnedObjectsV2/components/OwnedObject.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+    type SuiObjectResponse,
+    getObjectId,
+    getObjectDisplay,
+} from '@mysten/sui.js';
+
+import DisplayBox from '~/components/displaybox/DisplayBox';
+import { ObjectLink } from '~/ui/InternalLink';
+import { Text } from '~/ui/Text';
+import {
+    extractName,
+    parseImageURL,
+    parseObjectType,
+} from '~/utils/objectUtils';
+import { transformURL, trimStdLibPrefix } from '~/utils/stringUtils';
+
+type OwnedObjectTypes = {
+    obj: SuiObjectResponse;
+};
+
+function OwnedObject({ obj }: OwnedObjectTypes): JSX.Element {
+    const displayMeta = getObjectDisplay(obj).data;
+    const display = transformURL(parseImageURL(displayMeta));
+    return (
+        <div className="w-[50%] lg:flex lg:flex-wrap lg:justify-between">
+            <div className="my-2 flex h-fit w-full items-center truncate break-all sm:my-[1vh]">
+                <div className="mr-[20px] h-[60px] min-w-[60px] max-w-[60px]">
```

Comment:
Are we using tailwind now @plam-ml ? We should lean into that more whenever necessary. Leave a comment in Figma with me tagged so i have a visual grasp of what needs to be updated.

I have been slowly chipping away at every opportunity to align the Figma files with Tailwind progressively.

Reply:
@mystie711 Yep we are using tailwind. For example here is the list of margins https://v1.tailwindcss.com/docs/margin#app. The `60px` mark is between `3rem` and `4rem` 

I have to use `px` to `rem` conversion sometimes to check the sizing, so it would be easier for us if we include the `rem` instead of `px` in Figma also.

Reply:
Also, Figma does not yet offer this option @plam-ml. 

I have some other ideas for future iteration - token based design - which should help. For now, please tag me on Figma UI wherever we need to make a decision along with the github task that its related to. Will help us both make an informed decision quickly.